src/App.js

```diff
@@ -1,26 +1,199 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React in MateAcademy!
-        </a>
-      </header>
-    </div>
+import './fonts/stylesheet.css'
+import './css/App.css';
+
+import TodoItem from './Components/TodoItem';
+import Footer from './Components/Footer';
+
+class App extends React.Component{
+  constructor(){
+    super()
+    this.state = {
+      inputValue: '',
+      tasks: [],
+      activeTasks: [],
+      completedTasks: [],
+      item: 0,
+      clickOnFooter: false
+    }
+
+    this.addTask = this.addTask.bind(this)
+    this.inputTask = this.inputTask.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+    this.removeTask = this.removeTask.bind(this)
+    this.showActive = this.showActive.bind(this)
+    this.showCompleted = this.showCompleted.bind(this)
+    this.showAll = this.showAll.bind(this)
+    this.clearCompleted = this.clearCompleted.bind(this)
+    this.handleChangeFooter = this.handleChangeFooter.bind(this)
+
+
+  }
+
+  inputTask(event) {
+    this.setState({inputValue: event.target.value});
+  }
+
+  addTask(event) {
+    if(event.keyCode === 13){
+      const {inputValue, item} = this.state;
+      this.setState(prevState => {
+        if(inputValue === '')return;
```

Comment:
it should be the first string of this method. You don't need to execute anything if there is no value in the input